clj/src/cljd/core.cljd

```diff
@@ -8246,4 +8246,51 @@
       (doseq [tap @tapset]
         (try
           (tap x)
-          (catch Exception ex))))))
\ No newline at end of file
+          (catch Exception ex))))))
+
+(defprotocol Inst
+  (inst-ms* [inst]))
+
+(extend-type DateTime
+  IEquiv
+  (-equiv [o other]
+    (and (dart/is? other DateTime)
+         (== (.-millisecond o) (.-millisecond ^DateTime other))))
+
+  IComparable
+  (-compare [this other]
+    (if (dart/is? other DateTime)
+      (-compare (.-millisecond this) (.-millisecond ^DateTime other))
```

Comment:
you have native [https://api.flutter.dev/flutter/dart-core/DateTime/compareTo.html](compareTo) that can directly compare 2 dates

Reply:
+1 `compare` already works on `DateTime` because they are `Comparable`. Dart stdlib is better than JS stdlib (who said "which stdlib?")